### ISSUE TITLE
Error when clearing cache of app

### ISSUE THREAD

**Issue by idmarinas** [jaxon-php/jaxon-symfony#11]
Hello:

I'm getting this error when I use command `php bin/console cache:clear`

```bash
Script cache:clear returned with error code 1
  
   // Clearing the cache for the dev environment with debug                       
   // true                                                                        
  
  
  In Jaxon.php line 33:
                                                
    Warning: Undefined array key "SERVER_NAME"  
                                                
  
  cache:clear [--no-warmup] [--no-optional-warmers]
  
  
Script @auto-scripts was called via post-install-cmd
```

Usage:
Symfony Framework 5.2
PHP: 7.3

**Comment by feuzeu** [comment]
Hi,

The Jaxon class should not be instantiated when you invoke Symfony from cli. 
Why does that happen?

**Comment by idmarinas** [comment]
May be for a Twig Extension that I use for print js, css and script of Jaxon. 
This extension has a dependency injection of Jaxon\AjaxBundle\Jaxon.
Is always when add this dependency in extension when happens error.

**Comment by idmarinas** [comment]
Well, I have already fixed it, making the extension lazy.
With this everything works correctly, or for now not find problems.
This solution work for commands, but when you make test fail with `Symfony\Bundle\FrameworkBundle\Test\WebTestCase` and check if url work.

**Comment by feuzeu** [comment]
Hi,
It is fixed in the 3.2.1 release.
The `$_SERVER` data are used only when they are available.

**Comment by idmarinas** [comment]
Ok, good, thx
